### DISCUSSION TITLE
How to consider the viscous damping of floating platform under different motion

### DISCUSSION THREAD

**Discussion by YingjieDuan-hydrod** [OpenFAST/openfast#2086] [Q&A]
Good morning, I had a problem calculating the viscous properties of the platform. When floating platform is moving, the additional mass and damping caused by fluid viscosity change with the KC number. Whether openfast can consider the viscosity coefficient of different movements of the platform, or whether it needs to modify the code and add damping force that changes with the movement of the platform to the motion equation to consider the viscosity effect equivalent.
Thank you

**Comment by jjonkman** [comment]
Dear @YingjieDuan-hydrod,

HydroDyn does not currently support built-in KC-dependent drag coefficients.  It is up to the user, if they desire, to estimate the KC number for a given condition (sea state) and given hydrodynamic analysis node, and update the drag coefficients relevant to that KC number and node if desired.

FYI: In the newly started Phase I of the IEA Wind Task 56 (OC7) project, we are analyzing ways at better estimating viscous drag effects for various members of floating wind substructures, including KC-dependence.  We haven't yet decided if a modification to HydroDyn is warranted (e.g., to support KC dependence), but such a decision will be informed by the OC7 project.  If you have an interest in that topic, I encourage you to get involved (if you are not already).  

Best regards,
